src/asciidoc/docs/develop/portExistingCode.adoc

```diff
@@ -108,44 +110,78 @@ You need to do two things: set up a VCC to handle the user input, and consume th
 . Open the `frontend/index.html` file.
 +
-Notice the canvas element, with an `id` of `canvas`. You will target this element for the background color.
-. Open the `frontend/index.js` file, and add the following code to the top of the file, below the initial comment:
+Notice the canvas element, with an `id` of `canvas`.
+You will target this element for the background color.
+. Open the `frontend/index.js` file, and add the following code to the top of the file, below the initial comment.
 +
 [source,javascript]
 ----
-const Koji = require('@withkoji/vcc').default;
+const InstantRemixing = require('@withkoji/vcc').InstantRemixing;
+const instantRemixing = new InstantRemixing();
 
-const canvas = document.getElementById('canvas');
-if (canvas) {
-    canvas.style.background = Koji.config.settings.backgroundColor;
-}
+const upgradeBGColor = (bgcolor) => {
+  let canvas = document.getElementById('canvas');
+  if (canvas) {
+    canvas.style.backgroundColor = bgcolor;
+  }
+};
+
+// Set the initial bg color
+upgradeBGColor(instantRemixing.get(['settings', 'backgroundColor']));
+
+// Listen for updates
+instantRemixing.onValueChanged((path, newValue) => {
+  if (path[0] && path[0] === 'settings' && path[1] && path[1] === 'backgroundColor') {
+    upgradeBGColor(newValue);
+  }
+});
+
+let remixing = false;
+
+// Listen for remix mode
+instantRemixing.onSetRemixing((isRemixing) => {
+  remixing = isRemixing;
+});
+
+document.addEventListener('click', (event) => {
+    if (!remixing) return;
+    event.preventDefault();
+    instantRemixing.onPresentControl(['settings', 'backgroundColor']);
+});
+
+instantRemixing.ready();
 ----
 +
-This code imports the default export from the @withkoji/vcc package, finds the canvas element, and assigns a value that is pulled from the configuration files.
-. Refresh your application.
+This code instantiates InstantRemixing from the @withkoji/vcc package, gets the background color value from the customization file, finds the canvas element, and assigns the custom background color.
+It also monitors changes to the value and updates the background color automatically when a remixer selects a new value.
+Finally, it listens for remix mode and, if a user clicks the template while remixing, displays the backgroundColor VCC.
+. Refresh your preview.
 +
-You should now see the correct background color. If you return to the App settings customization, changing the background color should also update your preview automatically.
+You should now see the correct background color.
+If you return to the App settings customization, changing the background color should also update your preview automatically.
 
 == Publishing your template
 
-Because this project is compiled with Webpack, you must `require` the local JS files, instead of referencing them in the `index.html` file.
+Because this project is compiled with webpack, you must `require` the local JavaScript files, instead of referencing them in the `index.html` file.
```

Comment:
It might be worth noting here that the dev environment works correctly because the files already exist here. If they are not brought in using `require`, webpack won't know to include them in the bundle.